egs/wsj/s5/local/wsj_prepare_dict.sh

```diff
@@ -81,6 +81,13 @@ grep -v ';;;' $dir/cmudict/cmudict.0.7a | \
 (echo '!SIL SIL'; echo '<SPOKEN_NOISE> SPN'; echo '<UNK> SPN'; echo '<NOISE> NSN'; ) | \
  cat - $dir/lexicon1_raw_nosil.txt | sort | uniq > $dir/lexicon2_raw.txt || exit 1;
 
+# Axel Horndasch (axel.horndasch@fau.de):
+# This is probably the worst hack ever made; but it will help me test word classes
+# for Kaldi on the WSJ corpus; please forgive me, but some compounds from the word
+# class "US_STATE" like "WEST-VIRGINIA" were not in the lexicon ...
+cat $dir/lexicon2_raw.txt local/wclass/wsj_classes_lexicon.txt | sort | uniq > $dir/lexicon2_raw_tmp.txt
+mv $dir/lexicon2_raw_tmp.txt $dir/lexicon2_raw.txt
```

Comment:
The changes to 'egs/wsj/s5/local/wsj_prepare_dict.sh' and 'egs/wsj/s5/local/wsj_extend_dict.sh' are still in this commit. Please let me know if I should remove them (I might have to use other example word classes then; currently I use WEEKDAY and US_STATES).